dev/tools/gen_defaults/lib/filter_chip_template.dart

```diff
@@ -91,9 +124,7 @@ class _${blockName}DefaultsM3 extends ChipThemeData {
 
   @override
   IconThemeData? get iconTheme => IconThemeData(
-    color: isEnabled
-      ? ${color("$tokenGroup.with-icon.icon.color")}
-      : ${color("$tokenGroup.with-leading-icon.disabled.leading-icon.color")},
+    color: _leadingIconColor,
```

Comment:
```suggestion
    color: isEnabled
      ? isSelected
        ? ${color("$tokenGroup.with-leading-icon.selected.leading-icon.color")}
        : ${color("$tokenGroup.with-leading-icon.unselected.leading-icon.color")}
      : ${color("$tokenGroup.with-leading-icon.disabled.leading-icon.color")},
     
```

Reply:
I guess the nested ternary is more compact, we can change that, but I would keep the named getter, as it gives more meaning and it is reusable. For instance the input_chip_template reuses `_leadingIconColor` for the custom `iconTheme` icon color. What do you think?

Reply:
Named getters are not used in the templates. The format I suggested is consistency and readability. Templates are not about reusability as the tokens can be changed in the future. Please follow the suggestions.

Reply:
Ok, I removed the private getters. Thanks for the review!